content/en/about-filecoin/faq.md

```diff
@@ -124,7 +130,7 @@ Automatic repair of faulted data is a feature we've pushed off until after the m
 
 To avoid extortion, always ensure you store your data with a fairly decentralized set of storage providers (and note: it's pretty difficult for a storage provider to be sure they are the only person storing a particular piece of data, especially if you encrypt the data).
 
-Miners currently provide a 'dumb box' interface and will serve anyone any data they have. Maybe in the future, storage providers will offer ACLs and logins and such, but that requires that you trust the storage provider. The recommended (and safest) approach here is to encrypt data you don't want others to see yourself before storing it.
+Storage providers currently provide a 'dumb box' interface and will serve anyone any data they have. Maybe in the future, storage providers will offer ACLs (Access Control Lists) and logins and such, but that requires that you trust the storage provider. The recommended (and safest) approach here is to encrypt data you don't want others to see yourself before storing it.
```

Comment:
```suggestion
Storage providers currently provide a 'dumb box' interface and will serve anyone any data they have. Maybe in the future, storage providers will offer access control lists (ACLs) and logins and such, but that requires that you trust the storage provider. The recommended (and safest) approach here is to encrypt data you don't want others to see yourself before storing it.
```

When possible, try to use the full phrase before showing the acronym. See the contribution guide for details: https://docs.filecoin.io/about-filecoin/grammar-formatting-and-style/#acronyms